File: Modul4/Bahasa_Python/PRAK404-2310817210012-Nazmi_Hakim.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihanku = int(input("Masukkan Pilihan: "))

    if pilihanku == 5:
        print("Terimakasih, telah menggunakan kalkulator Nazmi Hakim")
        break
    elif pilihanku < 1 or pilihanku > 4:
        print("Input Anda salah, silakan coba lagi")
        continue

    nilai_pertama = float(input("Masukkan nilai pertama: "))
    nilai_kedua = float(input("Masukkan nilai kedua: "))

    if pilihanku == 1:
        hasiloperasi = nilai_pertama + nilai_kedua
        operasi = "Penjumlahan"
    elif pilihanku == 2:
        hasiloperasi = nilai_pertama - nilai_kedua
        operasi = "Pengurangan"
    elif pilihanku == 3:
        hasiloperasi = nilai_pertama * nilai_kedua
        operasi = "Perkalian"
    elif pilihanku == 4:
        if nilai_kedua == 0:
            hasiloperasi = 0
        operasi = "Pembagian"

    print(f"Hasil {operasi} antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasiloperasi:.2f}")

File: Modul4/Bahasa_C/PRAK404-2310817210012-Nazmi_Hakim.c
#include <stdio.h>
#include <string.h>

int main() {
    while (1) {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");

        int pilihan;
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Nazmi Hakim\n");
            break;
        } else if (pilihan < 1 || pilihan > 4) {
            printf("Input Anda salah, silakan coba lagi\n");
            continue;
        }

        double npertama, nkedua, hasil;
        printf("Masukkan nilai pertama: ");
        scanf("%lf", &npertama);
        printf("Masukkan nilai kedua: ");
        scanf("%lf", &nkedua);

        char operasi[20];
        if (pilihan == 1) {
            hasil = npertama + nkedua;
            strcpy(operasi, "Penjumlahan");
        } else if (pilihan == 2) {
            hasil = npertama - nkedua;
            strcpy(operasi, "Pengurangan");
        } else if (pilihan == 3) {
            hasil = npertama * nkedua;
            strcpy(operasi, "Perkalian");
        } else if (pilihan == 4) {
            if (nkedua == 0) {
                hasil = 0;
            }
            strcpy(operasi, "Pembagian");
        }

        printf("Hasil %s antara %.2lf dengan %.2lf adalah %.2lf\n", operasi, npertama, nkedua, hasil);
    }

    return 0;
}